wfdb/io/record.py

```diff
@@ -4373,7 +4377,7 @@ def wrsamp(record_name, fs, units, sig_name, p_signal=None, d_signal=None,
 
     """
     # Check for valid record name
-    if '.' in record_name: 
+    if '.' in str(record_name): 
```

Comment:
As directories might contain dots, maybe this wants to check for record_name.name instead.